Back-End/src/main/java/com/flight_search/dto/FlightDTO.java

```diff
@@ -0,0 +1,57 @@
+package com.flight_search.dto;
```

Comment:
flight_search
FlightDTO
there is no consistency in the files/directories naming